Update hyper-rustls dependency

Currently, it's impossible to specify such dependencies (even if, in my honest opinion, it's a bit cargo's fault):

```toml
[dependencies]
hyper = "*"
hyper-rustls = "*"
google-drive3 = "*"
```

because cargo will resolve `hyper-rustls v0.23.0` for the whole project, and `hyper-rustls v0.22.1` for google_drive3.

Then, it becomes impossible to write:

```rust
    let hub = google_drive3::DriveHub::new(
        hyper::Client::builder().build(
            hyper_rustls::HttpsConnectorBuilder::new()
                .with_native_roots()
                .https_only()
                .enable_http1()
                .build(),
        ),
        todo!()
    );
``` 
because it fails with: 
```
error[E0308]: mismatched types
 --> src/main.rs:4:13
  |
4 | /             hyper_rustls::HttpsConnectorBuilder::new()
5 | |                 .with_native_roots()
6 | |                 .https_only()
7 | |                 .enable_http1()
8 | |                 .build(),
  | |________________________^ expected struct `hyper_rustls::connector::HttpsConnector`, found struct `HttpsConnector`
  |
  = note: expected struct `hyper_rustls::connector::HttpsConnector<HttpConnector>`
             found struct `HttpsConnector<HttpConnector>`
  = note: perhaps two different versions of crate `hyper_rustls` are being used?

```

Maybe this dependency should be updated, or, like it was done [in this issue](https://github.com/Byron/google-apis-rs/issues/298), the `hyper` and the `hyper-rustls` crate should be exposed ?

Thank you very much

--- a/src/mako/api/lib.rs.mako
+++ b/src/mako/api/lib.rs.mako
@@ -43,7 +43,9 @@ ${lib.docs(c)}
 #[macro_use]
 extern crate serde_derive;
 
-extern crate hyper;
+// Re-export the hyper and hyper_rustls crate, they are required to build the hub
+pub extern crate hyper;
+pub extern crate hyper_rustls;
 extern crate serde;
 extern crate serde_json;
 // Re-export the yup_oauth2 crate, that is required to call some methods of the hub and the client

--- a/src/mako/api/lib/lib.mako
+++ b/src/mako/api/lib/lib.mako
@@ -176,8 +176,6 @@ To use this library, you would put the following lines into your `Cargo.toml` fi
 ```toml
 [dependencies]
 ${util.crate_name()} = "*"
-hyper = "^0.14"
-hyper-rustls = "^0.22"
 serde = "^1.0"
 serde_json = "^1.0"
 ```
@@ -245,7 +243,7 @@ Arguments will always be copied or cloned into the builder, to make them indepen
 ###############################################################################################
 <%def name="test_hub(hub_type, comments=True)">\
 use std::default::Default;
-use ${util.library_name()}::{${hub_type}, oauth2};
+use ${util.library_name()}::{${hub_type}, oauth2, hyper, hyper_rustls};
 
 % if comments:
 // Get an ApplicationSecret instance by some means. It contains the `client_id` and 

